cmd/kpod/pull.go

```diff
@@ -49,16 +50,13 @@ func pullCmd(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrapf(err, "could not get config")
 	}
-	store, err := getStore(config)
+
+	runtime, err := libpod.NewRuntime(libpod.WithStorageConfig(getStoreOptions(config)))
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "could not create runtime")
 	}
-
-	allTags := c.Bool("all-tags")
-
-	systemContext := common.GetSystemContext("")
-
-	err = libkpodimage.PullImage(store, image, allTags, false, systemContext)
+	runtime.SetOutput(os.Stdout)
```

Comment:
We should just pass a writer into the functions that need to output. Eventually we should be doing this with channels to track status, to make this easier to use headless.

Reply:
The CopyContext structure is actually used by containers/image, so we really can not change it easily.

Reply:
Hm. Have to think of a good way of handling this, then.

For now, I still think passing in a writer to the individual commands is a better option. I can see a few potential applications where we might want to throw most output to /dev/null but preserve progress bars for a few things